src/lib/theme/themeUtils.ts

```
@@ -1,7 +1,19 @@
-import { getContext } from "svelte";
 import { type ThemeConfig } from "$lib";
+import type { ClassValue } from "clsx";
+import { getContext } from "svelte";
 
 export function getTheme<K extends keyof ThemeConfig>(componentKey: K) {
   const theme = getContext<ThemeConfig>("theme");
   return theme?.[componentKey];
 }
+
+type Slots<T extends { slots: any }> = Omit<T["slots"], "base">
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)